community/webkitgtk/APKBUILD

```diff
@@ -1,7 +1,7 @@
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=webkitgtk
 pkgver=2.4.11
-pkgrel=3
+pkgrel=4
```

Comment:
I dont think we need bump pkgrel in this case as it only affects ppc64le which is not yet built. There are no changes that should affect any previously built packages.